client-js/queryEditor/QueryEditor.js

```diff
@@ -118,6 +119,16 @@ class QueryEditor extends React.Component {
     })
   }
 
+  formatQuery = () => {
+    // select * from table
+    const { query } = this.state
+    var result = sqlFormatter.format(query.queryText)
+    this.state.query.queryText = result
+    this.setState({
+      queryText: result
```

Comment:
For this call we'll want to call setState passing in the whole `query` object created. Since line 126 updates the `query` object, you can just pass that in here, like `this.setState({ queryText })`.

Reply:
yeah , i'm curious about line 126 and 127.
i know exactly that's repeated，but when delete line 126 or 127,
when i press alt+r,the sqlEditor content not change